app/Http/Controllers/TeamController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Team;
+use App\User;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Foundation\Auth\AuthenticatesUsers;
+use Illuminate\Http\Request;
+
+class TeamController extends Controller
+{
+	const ROLES = 'enum.user_roles';
+
+	public function __construct() {
+		$this->middleware('auth');
+	}
+
+	/**
+	 * Display a listing of the resource.
+	 *
+	 * @return \Illuminate\Http\Response
+	 */
+	public function index() {
+		//
+	}
+
+	/**
+	 * Show the form for creating a new resource.
+	 *
+	 * @return \Illuminate\Http\Response
+	 */
+	public function create() {
+		$students = User::students()->where('id', '!=', Auth::user()->id);
+
+		return view('teams.create', compact('students'));
+	}
+
+	/**
+	 * Store a newly created resource in storage.
+	 *
+	 * @param  \Illuminate\Http\Request  $request
+	 * @return \Illuminate\Http\Response
+	 */
+	public function store(Request $request) {
+		$attributes = request()->validate([
+			'name' => 'required',
+			'photo' => 'nullable',
```

Comment:
nitpick: le podemos llamar "image"? photo is short for photographs, whichs implies "taken with camera" 😄

Reply:
De hecho los nombres en esta parte del código tienen que ser los que me mandas del front-end. Ahorita me fijo en #94 para ver cuáles son los que estás poniendo y facilitar el trabajo a la hora de hacer la integración.